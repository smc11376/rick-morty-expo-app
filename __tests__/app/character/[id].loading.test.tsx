import { render, screen } from "@testing-library/react-native";
import CharacterDetailScreen from "@/app/character/[id]";

jest.mock("@/api/character", () => ({
  useGetCharacterById: () => ({
    character: undefined,
    isLoading: true,
  }),
}));

test("CharacterDetailScreen renders isLoading", () => {
  render(<CharacterDetailScreen />);

  expect(screen.toJSON()).toMatchSnapshot();
});
