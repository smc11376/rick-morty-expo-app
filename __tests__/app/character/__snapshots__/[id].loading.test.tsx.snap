// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterDetailScreen renders isLoading 1`] = `
<View
  style={
    {
      "backgroundColor": "rgb(60, 62, 68)",
      "flex": 1,
    }
  }
>
  <ActivityIndicator
    color="white"
    size="large"
    style={
      {
        "justifyContent": "center",
        "marginVertical": 50,
      }
    }
  />
</View>
`;
